lib/console/prompt_writer.ex

```diff
@@ -0,0 +1,41 @@
+defmodule TTT.PromptWriter do
+  alias TTT.Board
+  alias TTT.Messages
+
+  @column_divider " | "
```

Comment:
If these are used in a single place I'd inline them.
The names of the functions further down the line already have the names in them...